bump_version

```diff
@@ -70,6 +65,7 @@ pushd ${fedora_spec_temp_dir}
 # Split out the stuff before and after changelog
 csplit jellyfin-web.spec  "/^%changelog/" # produces xx00 xx01
 # Update the version in xx00
+new_version_sed="$( sed 's/\./\\./g' <<<"${new_version}" )" # Escape the '.' chars
```

Comment:
I am testing with `10.8.1-qqq`. RPM version cannot contain `-`. For DEB it will be `10.8.1~qqq`.
Since the escaping is unnecessary (not a regex), we have `new_version_sed` above (with `-.*` removed), and RPM wants a "clean" version, we need to remove this line.
```suggestion
```